### ISSUE TITLE
Wiki needs to be updated

### ISSUE THREAD

**Issue by James103** [yeelp/Scaling-Feast#113]
The last time the wiki was updated was in February of this year, and some of the pages have not even been updated since May/June of last year. Scaling Feast versions 1.6.x, 1.7.x, and 1.8.x have been released since then, with lots of new features.

* [ ] Add page for Tinker's mods integration
* [ ] Add page for Conarm integration
* [ ] Add "Exhausting Ore" page under "Blocks"
* [ ] Add "Exhausting Block" page under "Blocks" (may be redundant with above item)
* [ ] Add "Hearty Feast" page, also under "Blocks"
* [ ] Add "Exhausting Apple" page under "Items"
* [ ] Update the "Config" to support the rewrite and all new options added
* [ ] Add "HUD page", describing the GUI elements Scaling Feast adds
* [ ] Update "Spice of Life Module"
* [ ] Update "Spice of Life: Carrot Edition Module"

... and possibly more.

**Comment by yeelp** [comment]
Yeah absolutely. I had been working on the wiki at some point and got distracted by something (it was a bug or feature idea or something) and I think that's what turned into 1.8.x. Writing the wiki is a little difficult just due to the effort and content that needs to be checked, verified and added, along with screenshots and the like, but the procrastination is definitely on me.

The checklist definitely helps though!

**Comment by James103** [comment]
Last 3 wiki pages updated:
* Modules (Sep 3, 2021)
* Spice of Life Module (Sep 3, 2021)
* Config (February 2021)

All other pages were either updated in Q1 2021 or in the year 2020.

Will you please update the full wiki as soon as possible?
